templates/components/itilobject/actors/field.html.twig

```diff
@@ -54,16 +54,37 @@
    <select class="form-select" multiple="true" id="actor_{{ rand }}"
         {{ required ? 'required' : '' }}>
    {% for actor in actors %}
+      {% set unique_id = "user_opt_" ~ actortype ~ actor.itemtype ~ actor.items_id %}
       <option selected="true" value="{{ actor['itemtype'] ~ '_' ~ actor['items_id'] }}"
             data-itemtype="{{ actor['itemtype'] }}" data-items-id="{{ actor['items_id'] }}"
             data-use-notification="{{ actor['use_notification'] }}"
             data-alternative-email="{{ actor['alternative_email'] }}"
-            data-text="{{ actor['text']|verbatim_value }}" data-title="{{ actor['title']|verbatim_value }}">
+            data-text="{{ actor['text']|verbatim_value }}" data-title="{{ actor['title']|verbatim_value }}" data-glpi-popover-source="content{{ unique_id }}">
```

Comment:
Maybe this `data-glpi-popover-source` attribute should be placed only if popover is available.
To do so, you could probably generate popovers, store them into a variable, and print them after the select element.

```
{% set actors_popovers = [] %}
{% for actor in actors %}
    ....
    {% set popover %}
        popover code
    {% endset %}
    {% set actors_popovers = actors_popovers|merge([popover]) %}
{% endfor %}

{{ actors_popovers|join|raw }}
```